### PR TITLE
executor: add test case of admin check/checksum table in local temporary table

### LINKED ISSUE #25934: Forbid admin check table on local temporary tables

### LINKED ISSUE #25935: Forbid admin checksum table on local temporary tables

### PR DESCRIPTION
**Pull request by sylzd**
### What problem does this PR solve?

Issue Number: close #25934 
Issue Number: close #25935 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:
already support! I just add test case of local temporary.

### Check List 

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by djshow832 — APPROVED** [review]

File: executor/admin_test.go [hunk 1]
```diff
@@ -105,6 +105,28 @@ func (s *testSuite5) TestAdminCheckIndexInTemporaryMode(c *C) {
...
+
+	tk.MustExec("drop table if exists local_temporary_admin_checksum_table_with_index_test;")
+	tk.MustExec("drop table if exists local_temporary_admin_checksum_table_without_index_test;")
+	tk.MustExec("create temporary table local_temporary_admin_checksum_table_with_index_test (id int, count int, PRIMARY KEY(id), KEY(count))")
+	tk.MustExec("create temporary table local_temporary_admin_checksum_table_without_index_test (id int, count int, PRIMARY KEY(id))")
+	_, err = tk.Exec("admin checksum table local_temporary_admin_checksum_table_with_index_test;")
+	c.Assert(err.Error(), Equals, core.ErrOptOnTemporaryTable.GenWithStackByArgs("admin checksum table").Error())
+	_, err = tk.Exec("admin checksum table local_temporary_admin_checksum_table_without_index_test;")
+	c.Assert(err.Error(), Equals, core.ErrOptOnTemporaryTable.GenWithStackByArgs("admin checksum table").Error())
+	tk.MustExec("drop table if exists local_temporary_admin_checksum_table_with_index_test,local_temporary_admin_checksum_table_without_index_test;")
```
Comment (tiancaiamao) [inline executor/admin_test.go:127]: The title says `admin check index`

I see `admin check table` and `admin checksum table` 
... but where is the `admin check index`? 
https://github.com/pingcap/parser/blob/dd9b189324ced1cafb2ed35abf18c9d2037a614f/parser.y#L9722

Comment (sylzd) [reply executor/admin_test.go:127]: sorry, I thought `check table` contains index,  I fixed the title.

Comment (sylzd) [reply executor/admin_test.go:127]: check index wiil be added in https://github.com/pingcap/tidb/pull/26465/files

**Review by tiancaiamao — APPROVED** [review]

**Pull request merged**